.github/workflows/ci-test-pr.yml

```diff
@@ -0,0 +1,99 @@
+name: Build and Run Integration Test
+
+on:
+  push:
+    branches:
+      - '3875-get-env-var'
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  integration-test:
+    runs-on: ubuntu-latest
+
+    outputs:
+      image: ${{ steps.build-image.outputs.image }}
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
for this step and the next four, let's look up those actions on the github actions marketplace and update them to use the most recent version